### PR TITLE
docs: Add CONTRIBUTING.md

### LINKED ISSUE #3: Add a DEVELOPMENT.md/CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by justinprieto**
Add CONTRIBUTING.md.

Fixes #3

### CODE REVIEW

**Review by pjones-cedar — APPROVED** [review]
- [x] I fully understand what the code is doing and its context
- [x] The code is the best way to accomplish this problem (I would code it similarly)
- [ ] The code is well organized, easy to understand and matches Cedar design patterns and best practices
- [ ] The code handles edge cases (that I can think of) and has tests to validate them
- [ ] The appropriate people are reviewing (knowledge and security)
- [ ] I tested the code personally (Not required)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+Contributions are welcome. For all non-trivial changes, please create an issue first to help us track what is being worked on. PR titles should follow the [angular style](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits) for compatibility with our automated release tool.
```
Comment (juanrgon) [inline CONTRIBUTING.md:3]: Can you elaborate on this part more. It needs to be clear to the reader of this that:

- what the choices are
- how each choice iterates the version number
- how this automatically updates the CHANGELOG

Comment (justinprieto) [reply CONTRIBUTING.md]: ✅

**Review by pjones-cedar — DISMISSED** [review]
- [ ] I fully understand what the code is doing and its context
- [ ] The code is the best way to accomplish this problem (I would code it similarly)
- [ ] The code is well organized, easy to understand and matches Cedar design patterns and best practices
- [ ] The code handles edge cases (that I can think of) and has tests to validate them
- [ ] The appropriate people are reviewing (knowledge and security)
- [ ] I tested the code personally (Not required)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+No other change types will cause a release or be added to the changelog.
+
+## Local Development
+
+### Running tests
+
+First, make sure your images are up to date by running `docker-compose build`.
+
+The docker-compose setup can be used for local development and testing. It specifies two services, `local_development`, and `testing_environemnt`. `testing_environemnt` is used to run tests in CI. For fast iteration during local development, you can run the `local_development` container like so:
```
Comment (juanrgon) [inline CONTRIBUTING.md:18]:
```suggestion
The docker-compose setup can be used for local development and testing. It specifies two services, `local_development`, and `testing_environment`. `testing_environment` is used to run tests in CI. For fast iteration during local development, you can run the `local_development` container like so:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# Contributing
+
+Contributions are welcome. For all non-trivial changes, please create an issue first to help us track what is being worked on. PR titles should follow the angular style for compatibility with our automated release tool. See [here](https://www.conventionalcommits.org/en/v1.0.0/) for an explanation of the allowable commit message format.
+
+Breaking changes will cause the major version number of the library to be incremented. 
+`feat` changes will cause the minor version number to be incremented.
+`fix` and `perf` changes will cause the patch version number to be incremented.
+The commit messages for the foregoing will be automatically added to the changelog upon release.
```
Comment (juanrgon) [inline CONTRIBUTING.md:8]: So, commit messages on a branch are just added as-is to the changelog?

Comment (justinprieto) [reply CONTRIBUTING.md:8]: Correct

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Contributing
+
+Contributions are welcome. For all non-trivial changes, please create an issue first to help us track what is being worked on. PR titles should follow the angular style for compatibility with our automated release tool. See [here](https://www.conventionalcommits.org/en/v1.0.0/) for an explanation of the allowable commit message format.
```
Comment (juanrgon) [inline CONTRIBUTING.md:3]: The bit about PR title suggests that it will be used to trigger the release, but it actually doesn't
